src/map/mob.cpp

```diff
@@ -2403,14 +2400,14 @@ TIMER_FUNC(mob_respawn){
 	return 1;
 }
 
-void mob_log_damage(struct mob_data *md, struct block_list *src, int32 damage)
+void mob_log_damage(struct mob_data *md, struct block_list *src, int32 damage, int32 attdamage)
 {
 	uint32 char_id = 0;
 	int32 flag = MDLF_NORMAL;
 
 	if( damage < 0 )
```

Comment:
Do we need a similar check for attdamage?

Reply:
attdamage cannot be negative

Reply:
I'll add the check anyway, but just for safety, so nothing results in a substraction.